assets/js/just-track.js

```diff
@@ -1,4 +1,29 @@
+---
+---
```

Comment:
I'm wondering if we should add a comment to the front matter here and "explain" why its there. Seems like something we can forget and remove if we see it later on :)
Maybe even a link to the documentation and/or relevant information.